src/main.py

```diff
@@ -48,20 +48,6 @@
 sys.path.insert(1, comun.USERDAILIESDIR)
 
 
-def select_value_in_combo(combo, value):
-    model = combo.get_model()
-    for i, item in enumerate(model):
-        if value == item[1]:
-            combo.set_active(i)
-            return
-    combo.set_active(0)
-
-
-def get_selected_value_in_combo(combo):
-    model = combo.get_model()
-    return model.get_value(combo.get_active_iter(), 1)
-
-
```

Comment:
These two functions have been changed to suit the GtkTreeView widget and the new selection mechanism.
They are also moved inside the two class methods where they are referenced. No need to keep them as module functions.